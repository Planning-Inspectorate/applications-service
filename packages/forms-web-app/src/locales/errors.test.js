const enErrorsTranslations = require('./en/errors.json');
const cyErrorsTranslations = require('./cy/errors.json');

describe('locales/errors', () => {
	it('should return the correct English errors translations', () => {
		expect(enErrorsTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh errors translations', () => {
		expect(cyErrorsTranslations).toMatchSnapshot();
	});

	it('should return the same keys for English and Welsh translations', () => {
		expect(enErrorsTranslations).toHaveSameKeys(cyErrorsTranslations);
	});
});
