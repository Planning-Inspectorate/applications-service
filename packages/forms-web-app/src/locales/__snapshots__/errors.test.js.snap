// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`locales/errors should return the correct English errors translations 1`] = `
{
  "common.contactFormLinkText": "Fill out our contact form",
  "common.online": "Online:",
  "common.phone": "Phone:",
  "common.questions": "If you have any questions, you can contact us:",
  "common.serviceUnavailable": "Sorry, the service is unavailable",
  "haveYourSayJourney.paragraph1": "Try again later",
  "haveYourSayJourney.paragraph2": "We have not saved your answers.",
  "haveYourSayJourney.paragraph3": "When the service is available, you'll have to start again.",
  "haveYourSayJourney.title": "Sorry, there's a problem with the service",
  "pageNotFound.heading": "Page not found",
  "pageNotFound.paragraph1": "If you typed the web address, check it is correct.",
  "pageNotFound.paragraph2": "If you pasted the web address, check you copied the entire address.",
  "pageNotFound.paragraph3": "If the web address is correct or you selected a link or button, you can contact us:",
  "pageNotFound.title": "Page not found - Application service - GOV.UK",
  "unhandledException.paragraph1": "You'll be able to use the service again later.",
}
`;

exports[`locales/errors should return the correct Welsh errors translations 1`] = `
{
  "common.contactFormLinkText": "Llenwch ein ffurflen gyswllt",
  "common.online": "Ar-lein:",
  "common.phone": "Ffôn:",
  "common.questions": "Os oes gennych unrhyw gwestiynau, gallwch gysylltu â ni:",
  "common.serviceUnavailable": "Mae'n ddrwg gennym, nid yw'r gwasanaeth ar gael",
  "haveYourSayJourney.paragraph1": "Ceisiwch eto yn nes ymlaen.",
  "haveYourSayJourney.paragraph2": "Nid ydym wedi arbed eich atebion.",
  "haveYourSayJourney.paragraph3": "Pan fydd y gwasanaeth ar gael, bydd yn rhaid i chi ddechrau eto.",
  "haveYourSayJourney.title": "Mae'n ddrwg gennym, mae problem gyda'r gwasanaeth",
  "pageNotFound.heading": "Tudalen heb ei chanfod",
  "pageNotFound.paragraph1": "Os teipioch chi'r cyfeiriad gwe, gwiriwch ei fod yn gywir.",
  "pageNotFound.paragraph2": "Os gludoch chi'r cyfeiriad gwe, gwiriwch eich bod wedi copïo'r cyfeiriad cyfan.",
  "pageNotFound.paragraph3": "Os yw'r cyfeiriad gwe'n gywir neu os dewisoch ddolen neu fotwm, cysylltwch â Chymorth Cwsmeriaid yr Arolygiaeth Gynllunio:",
  "pageNotFound.title": "Tudalen heb ei chanfod",
  "unhandledException.paragraph1": "Byddwch yn gallu defnyddio'r gwasanaeth eto yn nes ymlaen.",
}
`;
